cloud-reactor-api/src/main/java/com/sequenceiq/cloudbreak/cloud/event/instance/StopStartDownscaleGetRunningInstancesResult.java

```diff
@@ -0,0 +1,44 @@
+package com.sequenceiq.cloudbreak.cloud.event.instance;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Set;
+
+import com.sequenceiq.cloudbreak.cloud.event.CloudPlatformResult;
+import com.sequenceiq.cloudbreak.cloud.model.CloudInstance;
+
+public class StopStartDownscaleGetRunningInstancesResult extends CloudPlatformResult {
+
+    private final List<CloudInstance> runningInstancesWithServicesNotRunning;
```

Comment:
Nit: Could this potentially be InstanceMetadata itself (probably an irrelevant comment given another one related to not using InstanceMetadata in this step)